### PR TITLE
Remove the nnfit executable and associated documentation

### PR DESCRIPTION
**Pull request by scarlehoff**
As per https://github.com/NNPDF/nnpdf/pull/1546#issuecomment-1118405570

At the moment just turning off the executable. In a later stage we can remove the code from the repo.

### CODE REVIEW

**Comment by scarlehoff** [comment]
Please have a look when you have time @Zaharid (or make someone else do so, given that you have physical access to everyone there in Santiago :P)

File: doc/sphinx/source/n3fit/methodology.rst [hunk 1]
```diff
@@ -40,8 +40,8 @@ The approach presented here inherits the technology developed by the NNPDF colla
 of fit pipeline but extends the possibility to test and improve fitting performance with modern
 techniques inspired by the deep learning community.
 
-The ``n3fit`` code is designed in python and one of its main goals is to replace the ``nnfit``
-program. It provides a simple abstraction layer which simplifies the life of developers when
+The ``n3fit`` code is designed in python and replaces the ``nnfit`` program used in the NNPDF3 family of fits.
```
Comment (RoyStegeman) [inline doc/sphinx/source/n3fit/methodology.rst:43]:
```suggestion
The ``n3fit`` code is designed in python and replaces the ``nnfit`` program used in the NNPDF3.X family of fits.
```

**Review by RoyStegeman — COMMENTED** [review]
Some mentions of `nnfit`  are still in the docs beyond the mentions that can stay such as an explanation of changes in `n3fit` wrt `nnfit` (though it might even be better to get rid of those as well since `nnfit`  was an internal naming convention anyway)

**Comment by scarlehoff** [comment]
> Some mentions of `nnfit` are still in the docs beyond the mentions that can stay such as an explanation of changes in `n3fit` wrt `nnfit` 

Could you point to the mentions you mean? (there are many so I'm not surprised if I've missed some, but I'm unable to find them?)

**Comment by RoyStegeman** [comment]
Let's wait with merging until #1546 can be merged.

**Comment by scarlehoff** [comment]
> Let's wait with merging until #1546 can be merged.

I don't know what's left of the other tbh, other than it being approved.

**Review by RoyStegeman — APPROVED** [review]

**Pull request merged**